chain/x/oracle/keeper/reporter.go

```diff
@@ -33,3 +33,22 @@ func (k Keeper) RemoveReporter(ctx sdk.Context, val sdk.ValAddress, addr sdk.Acc
 	ctx.KVStore(k.storeKey).Delete(types.ReporterStoreKey(val, addr))
 	return nil
 }
+
+// GetAllReportersOfValidator returns the reporter list of the given validator.
+func (k Keeper) GetAllReportersOfValidator(ctx sdk.Context, val sdk.ValAddress) (reporters []sdk.AccAddress) {
+	store := ctx.KVStore(k.storeKey)
+	storeKeyPrefix := append(types.ReporterStoreKeyPrefix, val.Bytes()...)
+	iterator := sdk.KVStorePrefixIterator(store, storeKeyPrefix)
+	defer iterator.Close()
+	for ; iterator.Valid(); iterator.Next() {
+		key := iterator.Key()
+		reporterBytes := key[21:]
```

Comment:
Can we use `1 + len(val)` or something like that? I prefer that more than hardcoding number 21.